src/app/dashboard/volunteers/page.js

```
@@ -0,0 +1,139 @@
+            <table className="table-auto w-full bg-white shadow-lg rounded-md overflow-hidden">
+                <thead className="bg-gray-100">
+                    <tr>
+                        <th className="px-4 py-2 text-left"> </th>
+                        <th className="px-4 py-2 text-left">Nom</th>
+                        <th className="px-4 py-2 text-left">Rôle</th>
+                        <th 
+                            onClick={() => setSortAsc(!sortAsc)} 
+                            style={{ cursor: 'pointer' }} 
+                            className="px-4 py-2 text-left"
+                        >
+                            Date d'adhésion
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)